test/integration/CommunityEndpoints.test.js

```diff
@@ -0,0 +1,246 @@
+import assert from 'assert'
+
+import { Contract, providers, utils, Wallet } from 'ethers'
+
+import StreamrClient from '../../src'
+import * as Token from '../../contracts/TestToken.json'
+
+import config from './config'
+
+function sleep(ms) {
+    return new Promise((resolve) => {
+        setTimeout(resolve, ms)
+    })
+}
+
+describe('CommunityEndPoints', () => {
+    let community
+
+    let testProvider
+    let adminClient
+    let adminWallet
+
+    beforeAll(async () => {
+        testProvider = new providers.JsonRpcProvider(config.ethereumServerUrl)
+        const network = await testProvider.getNetwork().catch((e) => {
+            throw new Error(`Connecting to Ethereum failed, config = ${JSON.stringify(config)}`, e)
```

Comment:
Unfortunately the second argument here does nothing. Should never suppress the original error message, can easily lead to debugging the wrong thing when original message if not suppressed would have revealed the problem immediately.

One option would be to change this into a log:

```js
console.error(`Connecting to Ethereum failed, config = ${JSON.stringify(config)}`)
throw e
```

Reply:
good catch. There are libraries that nest errors, but I'm not going to introduce one here